test/specs/commands/createTransactionCastVote.js

```diff
@@ -0,0 +1,232 @@
+											Then('it should get the inputs from sources using the Vorpal instance', then.itShouldGetTheInputsFromSourcesUsingTheVorpalInstance);
+											Then('it should get the inputs from sources using the passphrase source with a repeating prompt', then.itShouldGetTheInputsFromSourcesUsingThePassphraseSourceWithARepeatingPrompt);
+											Then('it should not get the inputs from sources using the second passphrase source', then.itShouldNotGetTheInputsFromSourcesUsingTheSecondPassphraseSource);
+											Then('it should not get the data using unvote source', then.itShouldNotGetTheDataUsingTheUnvoteSource);
+											Then('it should create a cast vote transaction with the passphrase, the second passphrase and the public keys prepended with a minus', then.itShouldCreateACastVoteTransactionWithThePassphraseTheSecondPassphraseAndThePublicKeysPrependedWithTheCorrectModifier);
+											Then('it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+										});
+									});
+									Given('an options object with vote set to "file:path/to/votes.txt" unvote set to "file:path/to/unvotes.txt"', given.anOptionsObjectWithVoteSetToAndUnvoteSetTo, () => {
+										Given('the votes can be retrieved from their sources', given.theVoteCanBeRetrievedFromItsSource, () => {
+											Given('the unvotes can be retrieved from their sources', given.theUnvoteCanBeRetrievedFromItsSource, () => {
+												When('the action is called with the options', when.theActionIsCalledWithTheOptions, () => {
+													Then('it should get the inputs from sources using the Vorpal instance', then.itShouldGetTheInputsFromSourcesUsingTheVorpalInstance);
+													Then('it should get the inputs from sources using the passphrase source with a repeating prompt', then.itShouldGetTheInputsFromSourcesUsingThePassphraseSourceWithARepeatingPrompt);
+													Then('it should not get the inputs from sources using the second passphrase source', then.itShouldNotGetTheInputsFromSourcesUsingTheSecondPassphraseSource);
+													Then('it should get the data using the unvote source', then.itShouldGetTheDataUsingTheUnvoteSource);
+													Then('it should create a cast vote transaction with the passphrase, the second passphrase and the public keys prepended with a minus', then.itShouldCreateACastVoteTransactionWithThePassphraseTheSecondPassphraseAndThePublicKeysPrependedWithTheCorrectModifier);
+													Then('it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+												});
+											});
+										});
+									});
+								});
+								Given('an options object with vote set to "file:path/to/votes.txt" and passphrase set to "file:path/to/my/passphrase.txt"', given.anOptionsObjectWithVoteSetToAndPassphraseSetTo, () => {
+									Given('the passphrase can be retrieved from its source', given.thePassphraseCanBeRetrievedFromItsSource, () => {
```

Comment:
This is redundant given the outer `Given the passphrase and second passphrase can be retrieved from their sources`